src/util-proto-name.c

```diff
@@ -60,10 +61,9 @@ void SCProtoNameInit()
             if (proto_ch == NULL)
                 continue;
 
-            int proto = atoi(proto_ch);
-            if (proto >= 255)
+            uint8_t proto;
+            if (StringParseU8RangeCheck(&proto, 10, 0, (const char *)proto_ch, 0, 255) < 0)
                 continue;
```

Comment:
should be 254 to be the same as before.
Or before the check was off by one ?

Reply:
Before, the check was off by one. Victor suggested to include 255 in the range.